mintpy/defaults/smallbaselineApp.cfg

```diff
@@ -119,9 +119,10 @@ mintpy.networkInversion.minRedundancy   = auto #[1-inf], auto for 1.0, min num_i
 
 ## Parallel processing with Dask for HPC
 mintpy.networkInversion.parallel    = auto #[yes / no], auto for no, parallel processing using dask
-mintpy.networkInversion.cluster	    = auto #[SLURM / PBS / LSF], auto for SLURM, cluster type for dask
-mintpy.networkInversion.numWorker   = auto #[int > 0], auto for 40, number of works for dask cluster to use
-mintpy.networkInversion.walltime    = auto #[HH:MM], auto for 00:40, walltime for dask workers
+mintpy.networkInversion.cluster     = auto #[SLURM / PBS / LSF], auto for SLURM, cluster type
+mintpy.networkInversion.config      = auto #[SLURM / PBS / LSF / no], auto for no (same as cluster), configuration name
+mintpy.networkInversion.numWorker   = auto #[int > 0], auto for 40, number of works to deploy
+mintpy.networkInversion.walltime    = auto #[HH:MM], auto for 00:40, walltime for each dask worker
```

Comment:
We need to be a bit careful here, because LSF takes wall time in HH:MM format, but SLURM and PBS take it in HH:MM:SS format. I would have to think about the best way to handle this.

Reply:
`mintpy.objects.cluster.get_cluster()` sounds like a good option.

Reply:
Yes. That is probably the best place for it. I guess I'll just check the length of string and the cluster type and remove/append the ':SS' part as needed. 

So if its `LSF` but gives an 8 character wall time, I can remove the last 3 characters to make it 'HH:MM' format, or if its `SLURM` but gives a 5 character wall time, I can append ':00' to the end to force 'HH:MM:SS' format. Thoughts on this strategy?

Reply:
Your way sounds good to me.